hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/ContainersResponse.java

```diff
@@ -60,16 +61,24 @@ public static class ContainersResponseData {
     @JsonProperty("totalCount")
     private long totalCount;
 
+    /**
+     * prevKey will be the last key of the previous page.
+     */
+    @JsonProperty("prevKey")
+    private long prevKey;
+
```

Comment:
in this class you reference it as `prevKey` but in the `ContainerEndpoint` when you are passing it you say `lastContainerID` (also in the jira description is says it this way)

https://github.com/apache/ozone/blob/d2c3b4aad073f9b28ee30f3c80cf12c6f4247326/hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/ContainerEndpoint.java#L161-L163

shouldn't we use `lastContainerID` here as well? `prevKey` seems a bit hard to understand at first, or maybe just document the change better. 
let me know what you think!

Reply:
Thanks for the comment @dombizita !

There are two reasons I chose to use the name `prevKey` :- 

1. The response object's `prevKey` will be reused by the same API as the previous key for retrieving the next set of records, enabling pagination. Since the query parameter in the ContainerEndpoint is also named "prevKey," I decided to maintain consistency and keep the name as "prevKey."

2. `prevKey` is also used as the name in other APIs of Recon for pagination. By maintaining consistent naming conventions across the codebase, we can prevent confusion among developers.

Overall, these reasons support my decision to keep the variable named as "prevKey."

Reply:
thanks for the explanation @ArafatKhan2198, it makes sense to leave it as it is!